feature/gnoi/system/tests/ping_test/ping_test.go

```diff
@@ -110,7 +111,9 @@ func TestGNOIPing(t *testing.T) {
 	if len(ipv6Addrs) == 0 {
 		t.Fatalf("Failed to get a valid IPv6 loopback address: %+v", ipv6Addrs)
 	}
-
+	if *deviations.ExplicitInterfaceInDefaultVRF {
+		fptest.AssignToNetworkInstance(t, dut, lbIntf, "mgmt", 0)
```

Comment:
"mgmt" should be the default VRF? This test case currently does not test ping from non-default VRF, right?

Reply:
It could be default VRF if your connection entrypoint is via default VRF and the gNOI server is operational in that VRF. In my lab I'm using 'mgmt', but it's a variable, if in your bindings/textproto file your connection IP to gNOI is through the default VRF, I can change this to be default. The test-case simply pings from the obtained connection endpoint, I think it doesn't look at the NI, it's just which NI the client connected from

Reply:
I think there are 2 moving parts: the gNOI wanted behavior (in terms of specifying the VRF) and different OS implementations. If the source IP is not specified, I do feel like that the current gNOI seems allow different OS implementation choose their own VRF or source IP. 

This test currently only focuses on ping in the default VRF. Maybe let it keep it that way?  

BTW, `ExplicitInterfaceInDefaultVRF` should be only about the adding interface in the default VRF, not arbitrary VRF.

Reply:
changed to default VRF